src/h5web/vis-packs/core/heatmap/config.tsx

```diff
@@ -84,18 +79,8 @@ function initialiseStore(onRehydrated: () => void) {
   );
 }
 
-const context = createContext<UseStore<HeatmapConfig> | undefined>(undefined);
-const { Provider } = context;
-
-export function useHeatmapConfig<U>(
-  selector: StateSelector<HeatmapConfig, U>,
-  equalityFn?: EqualityChecker<U>
-): U {
-  const useStore = useContext(context);
-  assertDefined(useStore);
-
-  return useStore(selector, equalityFn);
-}
+const { Provider, useStore } = createContext<HeatmapConfig>();
+export const useHeatmapConfig = useStore;
 
 // https://github.com/pmndrs/zustand/issues/128#issuecomment-673398578
 export function HeatmapConfigProvider(props: ConfigProviderProps) {
```

Comment:
Note that this wrapper is still necessary because of the asynchronous rehydration of the store by the `persist` middleware. https://github.com/pmndrs/zustand/issues/346